src/main/java/cz/mzk/utils/SolrUtils.java

```diff
@@ -9,4 +10,13 @@
     public static void setModify(final SolrInputDocument inputDocument, final String fieldKey, final Object fieldValue) {
         inputDocument.addField(fieldKey, Collections.singletonMap("set", fieldValue));
     }
+
+    public static String wrapQueryStrForEmptyValue(String fieldName) {
+        return "-" + fieldName + ":[\"\" TO * ]";
```

Comment:
nerozumim co to je :) pokud chces query na prazdne pole pouzivej "-fieldName:*"

Reply:
Problem je v tom, ze v solri sa nachadza vzdy(vacsia cast) issn filed. Je tam nastavena issn:"", co pomocou -issn:* nenajdem, lebo toto predpoklada, ze tam nieje ziaden issn field.
Cize budem musiet kontrolovat, ci sa tam field issn nachadza + ci issn field je prazdy.
A tento prikaz ["" TO *], ak som dobre pochopil, riesi to, ze mi kontroluje hodnotu "" pre vsetky dokumenty.

Reply:
Napisal som ti spravu, ak budes mat cas pozrel by si sa na nu prosim. :)